charts/configcat-proxy/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: policy/v1
```

Comment:
Max 1 pod can be down at a time.
Its a silly thing but if when we recycle our nodes; we don't want the node recycler to recycle a node with this pod on it unless more than 1 pod is available.